tests/testthat/payload/validate_anc_payload.json

```diff
@@ -1,5 +1,6 @@
 {
   "type": "anc",
   "path": "testdata/anc.csv",
-  "shape": "testdata/malawi.geojson"
+  "shape": "testdata/malawi.geojson",
+  "originalFilename": "original.csv"
```

Comment:
It would actually be super useful to have the hash come through as well.  This means that there are 3 pieces of information we're using to describe a file:

* `hash` - its md5, as computed by hint
* `path` - where hint has put the file on the shared volume
* `originalFilename` (or just `filename`) - the filename as uploaded by the user

These 3 pieces could go together in one little schema and be passed around as a dict.

Otherwise when we start on the caching layer the first thing we do will start hashing the file :)